src/ZkFold/Symbolic/Compiler/ArithmeticCircuit/Internal.hs

```diff
@@ -73,9 +71,8 @@ data ArithmeticCircuit a i o = ArithmeticCircuit
 deriving instance (NFData a, NFData (o (Var i)), NFData (Rep i))
     => NFData (ArithmeticCircuit a i o)
 
--- | A finite field of a large order.
--- It is used in the compiler for generating new variable indices.
-type VarField = Zp BLS12_381_Scalar
+-- | Variables are SHA256 digests (32 bytes)
+type VarField = Zp (2 ^ (32 * 8))
```

Comment:
Is `VarField` still in use? It looks like some code referring to it was removed.